config/v1/types_ingress.go

```diff
@@ -90,8 +90,28 @@ type IngressSpec struct {
 	// provider of the current cluster and are required for Ingress Controller to work on OpenShift.
 	// +optional
 	LoadBalancer LoadBalancer `json:"loadbalancer,omitempty"`
+
+	// logLevel describes the desired logging verbosity for the Ingress Operator
+	// Any one of the following values may be specified:
+	// * Normal
+	// * Debug
```

Comment:
Vaild values are: "Normal", "Debug".
BTW are you going to add only two levels ?
What log package are you going to use ?

Reply:
The ingress operator uses [zapr](https://github.com/go-logr/zapr) for logging. If the "Normal" level maps to the existing default log level, then Normal is info, and there's only one formal log level more verbose than that in zapr: debug.

That said, it looks like you can add more verbose log levels, they just won't have names, so I'll add one for "Trace" log level